Add 2022 community update (#1789)
writethedocs/www: docs/blog/2022-community-update.rst

.. post:: Aug 22, 2022
   :tags: stats, newsletter
   :author: Eric Holscher

Write the Docs Community Update - August 2022
=============================================

Hey there documentarians!

Eric Holscher here, one of the co-founders of Write the Docs.
Filling in again for the newsletter team in August this year,

> we have a few updates from the community and what has been going on.

Reviewer: ```suggestion
we have a few updates from the community about what has been going on.
```

Thanks again to `Aaron <https://www.aaronrcollier.com/>`_ for taking over the newsletter editor role,

> and to `Beth <https://bethaitman.com/>`_ for doing it for the last few years.

Reviewer: ```suggestion
and thanks to `Beth <https://bethaitman.com/>`_ for doing it for the last few years.
```

> We're looking forward to seeing everyone online for our virtual `Prague <http://www.writethedocs.org/conf/prague/2022/>`_ on **September 11-13** in the CEST timezone.

Reviewer: ```suggestion
We're looking forward to seeing everyone online for our virtual `Prague <http://www.writethedocs.org/conf/prague/2022/>`_ on **September 11-13** in the CEST timezone. Tickets are still on sale until a week before the event, so grab yours now if you haven't yet.
```

> We're also getting excited about our upcoming virtual `Australia & India <http://www.writethedocs.org/conf/australia/2022/>`_ conference on **December 8-9** in the AEDT timezone.

Reviewer 2: ```suggestion
We're also getting excited about our upcoming virtual `Australia <http://www.writethedocs.org/conf/australia/2022/>`_ conference on **December 8-9** in the AEDT timezone.
```

Author: Thought I caught that 🙃

> The talks were just announced for the Prague conference, and the `Australia conference CFP <https://www.writethedocs.org/conf/australia/2022/cfp/>`_ **is open until September 16 if you have ideas to present.**

Reviewer: ```suggestion
The `Australia conference CFP <https://www.writethedocs.org/conf/australia/2022/cfp/>`_ **is open until September 16 if you have ideas to present.**
```

Reviewer: Don't think we need to repeat that the talks for Prague are available since it's already 2 blog posts ago, and might take away from the signposting of the open CFP for AUS.

The `videos <https://www.youtube.com/playlist?list=PLZAeFn6dfHpnDhFvXG8GprqlLlzSQRBui>`_ from the conference are available online.

2023 Conferences
----------------

We are looking forward to the rest of our 2022 conferences,
but we're also starting to plan for 2023.
We had hoped to do a hybrid event last year,
but found the logistics quite difficult to figure out.
You can read more in our `2022 virtual announcement <https://www.writethedocs.org/conf/portland/2022/news/conference-going-virtual/>`_ about the difficulties we had with hybrid.

We know the community is slowly becoming more excited for in-person events as time goes on.
The hope is to have events in 2023 that are valuable for both our in-person and virtual attendees,
but we haven't figured out exactly what that looks like.

We continue to work to balance the bandwidth of our organizing team,
the accessibility that virtual provides,
and the value from live human interaction that we all miss.
We will have more announcements about our 2023 plans closer to the end of the year,
but we appreciate all the support we've gotten from you all as we figure things out.

> 

Reviewer: Do we want to promote the new team org structure? We published it already but it can't hurt to socialize the opportunities to join teams or move around between teams...

Slack update
------------

> Slack has `changed it policy around free plans <https://slack.com/blog/news/pricing-and-plan-updates>`_,

Reviewer 3: Do we want to add a sentence that we intentionally use the free plan, in case this triggers another flood of "why don't you ask for sponsorship"? Might be too much detail.

Reviewer 4: Could borrow the admonition from https://www.writethedocs.org/slack/#slack-history

Author: Yea, I could at least link to it.

starting in September we will have **3 months of Slack history**.
This will make the community a much more useful place for people who interact occasionally,
allowing them to catch up on past conversations a bit more easily. 

> We also wanted to take the time to show appreciation to our `Slack moderation team <https://www.writethedocs.org/team/#slack-moderation>`_.

Reviewer: Do we want to specifically shoutout to Janine, or did we do that already somewhere else?

We are overjoyed at the helpful community that we've been able to curate on our Slack network,
and they play a huge role in that.
If you aren't on our Slack,
we hope you'll join us and contribute to the many varied discussions happening each day.

Salary Survey
-------------

Our annual salary survey gathers data about salaries for people working in documentation and related fields around the world.

> `Results of the third salary survey <https://www.writethedocs.org/surveys/salary-survey/2021/>`__ are available on our website.

Reviewer: ```suggestion
`Results of the 2021 salary survey <https://www.writethedocs.org/surveys/salary-survey/2021/>`__ are available on our website.
```

> We continue to hear from members of the community that use this information to set expectations around salary with their employers.

Reviewer: ```suggestion
We continue to receive great feedback from members of the community that use this information to set expectations around salary with their employers.
```

Reviewer 2: s/hear/receive/ ?

Leveling the playing field in employment negotiations is one of the best things we can do to help our community members,

> and plan to continue working on this endeavour. 

Reviewer: ```suggestion
and we plan to continue working on this endeavor. 
```

We're hoping to launch next year's survey here soon,
but haven't put the finishing touches on it quite yet.

> Watch the newsletter for that in the coming months.

Reviewer: ```suggestion
Watch the newsletter for updates in the coming months.
```
